CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ x.x.x Release notes (yyyy-MM-dd)
 * Added support for opening Realm in read-only mode. ([#260](https://github.com/realm/realm-dart/pull/260))
 * Primary key fields no longer required to be `final` in data model classes ([#240](https://github.com/realm/realm-dart/pull/240))
 * List fields no longer required to be `final` in data model classes. ([#253](https://github.com/realm/realm-dart/pull/253))
+* Generated classes no longer use `List` class on interface, but instead use `RealmList`. This allow user to avoid casting, when accessing `RealmList` specific functionality. ([#270](https://github.com/realm/realm-dart/pull/270))
```

Comment:
I am sorry, should have said it clearer. I suggest we have this something like this

### Internal
 * Use `RealmList` instead of `List` in generated Realm object classes. It enables access to a Realm list specific APIs.

Reply:
Fixed